pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
This file is auto generated using the settings I was passing in on the command line.